src/inmemoryfile.cpp

```diff
@@ -0,0 +1,63 @@
+#include "inmemoryfile.hpp"
```

Comment:
files names convention? 🤔

Reply:
what do you mean by that, my friend?

Reply:
other files names -> snake_case
<img width="258" alt="Zrzut ekranu 2019-07-25 o 10 35 30" src="https://user-images.githubusercontent.com/11761319/61859275-1a554980-aec8-11e9-8fc6-78939ed55f50.png">